app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+    has_secure_password
+    validates :username, presence: true, uniqueness: true
+    validates :email, presence: true, uniqueness: true
+    validates :password, presence: true
+    validates :bio, length: {maximum: 150}
+    validates :location, length: {maximum: 50}
```

Comment:
Length is unnecessary. You can remove these

Reply:
removed the length part along with Rspec test part in user_spec.rb